packages/journey-client/src/lib/fr-webauthn/script-parser.ts

```
@@ -0,0 +1,192 @@
+  // e.g. `rp: {\n id: \"https://user.example.com:3002\",\n name: \"ForgeRock\"\n }`
+  const rp = getIndexOne(text.match(/rp"{0,}:\s{0,}{([^}]+)}/)).trim();
+  // e.g. `id: \"example.com\"
+  const rpId = getIndexOne(rp.match(/id"{0,}:\s{0,}"([^"]*)"/));
+  // e.g. `name: \"ForgeRock\"`
+  const rpName = getIndexOne(rp.match(/name"{0,}:\s{0,}"([^"]*)"/));
+
+  // e.g. `user: {\n id: Uint8Array.from(\"NTdhN...RiNjI5\",
+  // function (c) { return c.charCodeAt(0) }),\n
+  // name: \"57a5b4e4-...-a4f2e5d4b629\",\n
+  // displayName: \"57a5b4e4-...-a4f2e5d4b629\"\n }`
+  const user = getIndexOne(text.match(/user"{0,}:\s{0,}{([^]{0,})},/)).trim();
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)